src/applications/static-pages/health-care-manage-benefits/components/CernerCallToAction/index.js

```diff
@@ -0,0 +1,146 @@
+      facilities: [],
+      fetching: true,
+    };
+  }
+
+  componentDidMount() {
+    const { facilities } = this.props;
+
+    // Escape early if there are no facilities.
+    if (isEmpty(facilities)) {
+      return;
+    }
+
+    // Derive the cerner facilities.
+    const cernerFacilities = facilities.filter(facility => facility.isCerner);
+
+    // Escape early if there are no cerner facilities.
+    if (isEmpty(cernerFacilities)) {
+      // WARNING: Add sentry logging here if there are no cerner facilities found, this should never happen as the component only renders when there ARE cerner facilities.
+      return;
+    }
+
+    // Derive the list of facility IDs.
+    const cernerFacilityIDs = cernerFacilities.map(
+      facility => `vha_${facility.facilityId}`,
+    );
+
+    // Fetch cerner facilities.
+    this.fetchFacilities(cernerFacilityIDs);
+  }
+
+  fetchFacilities = async facilityIDs => {
+    // Show loading state.
+    this.setState({ fetching: true });
+
+    try {
+      // Fetch facilities and store them in state.
+      const response = await apiRequest(
+        `${environment.API_URL}/v1/facilities/va?ids=${facilityIDs.join(',')}`,
+      );
+      this.setState({ facilities: response?.data, fetching: false });
+
+      // Log any API errors.
+    } catch (error) {
+      this.setState({ error: error.message, fetching: false });
+    }
+  };
+
+  render() {
+    const { text, type } = this.props;
+    const { error, fetching, facilities } = this.state;
+
+    // Escape early if we are fetching.
+    if (fetching) {
+      return <LoadingIndicator message="Loading your information..." />;
+    }
+
+    // Escape early if there was an error fetching the Cerner facilities.
+    if (error || isEmpty(facilities)) {
+      return (
+        <AlertBox
+          headline="Something went wrong"
+          content="We’re sorry. Something went wrong on our end. Please try again later."
+          status="error"
+        />
+      );
+    }
+
+    // Derive the Cerner facility names.
+    const facilityNames = facilities.map(
+      facility => facility?.attributes?.name || 'unknown facility name',
+    );
+    const joinedFacilityNames =
+      facilityNames.join(', ') || 'unknown cerner facility(s)';
+
+    return (
+      <div className="usa-alert usa-alert-warning">
+        <div className="usa-alert-body">
+          <h3 className="usa-alert-heading">
+            According to our records, you are registered at a clinic within{' '}
+            {joinedFacilityNames}. VA providers at this facility and its clinics
+            are using the new My VA Health portal.
+          </h3>
+          <p className="usa-alert-text vads-u-margin-y--4">{text}</p>
+          <h3 className="usa-alert-heading">
+            View {type} from {joinedFacilityNames}
+          </h3>
+          <a
+            className="usa-button"
+            href="https://ehrm-va-test.patientportal.us.healtheintent.com/pages/health_record/results/labs"
```

Comment:
Is there a constant anywhere that we could import here? I have no idea what this URL is lol.

Reply:
haha great question! Maybe? I can make it a constant but it seems to be different for each page...

![image](https://user-images.githubusercontent.com/12773166/86057409-5ec46000-ba1c-11ea-9d32-2b4eac08b4ca.png)

![image](https://user-images.githubusercontent.com/12773166/86057442-6ab02200-ba1c-11ea-85f1-2946bdb14529.png)

Reply:
It looks like the `ehrm` domain is used a couple other times in code - in VAOS and the dashboard. Would you mind making it a constant? 

example - `src/applications/vaos/utils/appointment.js`, `src/applications/personalization/dashboard/components/ManageYourVAHealthCare.jsx`

Something like -

```js
// platform/hm/where/cerner

const root =  environment.isProduction()
          ? 'https://patientportal.myhealth.va.gov/'
          : 'https://ehrm-va-test.patientportal.us.healtheintent.com/';

export default getCernerUrl(path) {
    return `${root}${path}`;
}
```

I'm not sure exactly, even on the naming convention - whether it should it be Cerner, or the public term we use - My VA Health

Reply:
Not at all! 1 constant coming right up 🙂